AsusSMC/AsusSMC.cpp

```diff
@@ -163,6 +163,38 @@ OSDictionary *AsusSMC::getDictByUUID(const char *guid) {
 
 OSDefineMetaClassAndStructors(AsusSMC, IOService)
 
+
+#define kIOPMPowerOff                       0
+#define kNumberOfStates                     2
+
+static IOPMPowerState powerStates[kNumberOfStates] = {
+   {1, kIOPMPowerOff, kIOPMPowerOff, kIOPMPowerOff, 0, 0, 0, 0, 0, 0, 0, 0},
+   {1, kIOPMPowerOn, kIOPMPowerOn, kIOPMPowerOn, 0, 0, 0, 0, 0, 0, 0, 0}
+};
+
+IOReturn AsusSMC::setPowerState(unsigned long powerStateOrdinal, IOService * whatDevice) {
+
+    if (powerStateOrdinal == kIOPMPowerOff) {
+        DBGLOG("atk", "Power off");
+        /**
+        *  Sleep, shutdown or reboot detected
+        */
+        setKBLLevel(kbl_level, false, false);
+    } else {
+        DBGLOG("atk", "Waking up");
+        kbl_level = readKBBacklightFromNVRAM();
+        setKBLLevel(kbl_level);
```

Comment:
Please change this to `setKBLLevel(kbl_level, false, false);`

Reply:
done, 

also fix a copy-paste mistake that I made on power off, the level is now set explicitly to 0